src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.html

```diff
@@ -1,4 +1,147 @@
+        </div>
+      </dl>
+      <dl class="govuk-summary-list" *ngIf="hearingActualsMainModel.hearingActuals.hearingOutcome.hearingResultReasonType.length > 0">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Select a reason for this result</dt>
+          <dd class="govuk-summary-list__value">{{hearingActualsMainModel.hearingActuals.hearingOutcome.hearingResultReasonType}}</dd>
+        </div>
+      </dl>
+      <br>
+      <h1 class="govuk-heading-m">Hearing timings</h1>
+      <span class="govuk-body align-right">
+        <a href="javascript:void(0)" [routerLink]="['/', 'hearing-actual', 'actuals-date-and-time']" class="govuk-link">Update</a>
+      </span>
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Start time</dt>
+          <dd class="govuk-summary-list__value">{{hearingActualsMainModel.hearingActuals.actualHearingDays[0].hearingStartTime | date:"HH:mm" }}</dd>
+        </div>
+      </dl>
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Finish time</dt>
+          <dd class="govuk-summary-list__value">{{hearingActualsMainModel.hearingActuals.actualHearingDays[0].hearingEndTime | date:"HH:mm" }}</dd>
+        </div>
+      </dl>
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Pause time</dt>
+          <dd class="govuk-summary-list__value" *ngIf="hearingActualsMainModel.hearingActuals.actualHearingDays[0].pauseDateTimes.length > 0">
+            {{hearingActualsMainModel.hearingActuals.actualHearingDays[0].pauseDateTimes[0].pauseStartTime}}
+          </dd>
+        </div>
+      </dl>
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Resume time</dt>
+          <dd class="govuk-summary-list__value" *ngIf="hearingActualsMainModel.hearingActuals.actualHearingDays[0].pauseDateTimes.length > 0">
+            {{hearingActualsMainModel.hearingActuals.actualHearingDays[0].pauseDateTimes[0].pauseEndTime}}
+          </dd>
+        </div>
+      </dl>
+      <br>
+      <h1 class="govuk-heading-m">Participants</h1>
+      <span class="govuk-body align-right">
+        <a href="javascript:void(0)" [routerLink]="['/', 'hearing-actual', 'actuals-parties']" class="govuk-link">Update</a>
+      </span>
+      <div *ngFor="let actualHearingDay of hearingActualsMainModel.hearingActuals.actualHearingDays">
+        <dl class="govuk-summary-list">
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">
+              {{actualHearingDay.actualDayParties[0].actualIndividualDetails.firstName}} {{actualHearingDay.actualDayParties[0].actualIndividualDetails.lastName}}
+              <br>
```

Comment:
if this actualHearingDay.actualDayParties[0] can be put into the typescript variable as this is used loads of time in the template?

Reply:
Good suggestion @rayliangatos, I have now made the changes accordingly.